src/widgets/state-widget/Badge.tsx

```diff
@@ -0,0 +1,29 @@
+import { View } from '@adobe/react-spectrum';
+import { ViewProps } from '@react-types/view';
+import { ReactNode } from 'react';
+
+export interface BadgeProps {
+	label: string;
+
+	backgroundColor: ViewProps['backgroundColor'];
+}
+
+export function Badge({
+	special,
+	children
+}: {
+	special: boolean;
+	children: ReactNode;
+}): JSX.Element {
```

Comment:
```suggestion
}} {
```
Please remove the explicit return type of the component. :wink: 
See https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components for more information.

Reply:
I tried applying your suggested change and I got an error (Unexpected token, expected "," (17:1)). Then I also get this error: 'Badge', which lacks return-type annotation, implicitly has an 'any' return type.ts(7010) and I am not sure what I have to do now. What can I do to fix this problem or is there anything I can do to get it working with your suggested change?

Reply:
@Sofia-Rommel Could you quickly jump into Discord?